koku/reporting/rate/models.py

```diff
@@ -15,37 +15,31 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-"""Models for identity and access management."""
+"""Models for rates."""
 from uuid import uuid4
 
+from django.contrib.postgres.fields import JSONField
 from django.db import models
 
-TIMEUNITS = (('nil', 'nil'),
-             ('hour', 'hour'),
-             ('minute', 'minute'),
-             ('second', 'second'),
-             ('day', 'day'),
-             ('month', 'month'),
-             ('year', 'year'),
-             ('onetime', 'onetime'))
-
 
 class Rate(models.Model):
-    """A rate for calculating costs.
+    """A rate for calculating charge.
 
-    A rate is (price * metric_usage / timeunit).
+    Support various types of rates (flat, fixed, tiered, discount).
     """
 
+    METRIC_CPU_CORE_HOUR = 'cpu_core_per_hour'
+    METRIC_MEM_BYTES_HOUR = 'memory_bytes_per_hour'
```

Comment:
Yes, it certainly will.  I'm working on updating things on the masu side now.  One question.  Should this be `memory_gb_per_hour` since we are calculating the charge based on GB-hour?

Reply:
I'm flexible, up to you guys

Reply:
Made an update for `memory_gb_per_hour` for now, since we were already using `core` instead of `milicore`.